src/config/fragments.rs

```diff
@@ -77,6 +77,9 @@ pub(crate) struct UpdateFleetLock {
 pub(crate) struct UpdatePeriodic {
     /// A weekly window.
     pub(crate) window: Option<Vec<UpdatePeriodicWindow>>,
+    /// Whether to use system time's time zone (i.e. `/etc/localtime`)
+    /// as the time zone for reboot windows.
+    pub(crate) use_system_time: Option<bool>,
```

Comment:
Bikeshedding: I think this may work better has a `use_utc: Option<bool>` which is on by default, and users have to explicitly opt-out.